parser/testdata/workflows/greetings-v08-spec.sw.yaml

```diff
@@ -84,6 +85,7 @@ states:
         total: PT1S
         single: PT2S
     numCompleted: 13
+    transition: CheckVisaStatusSwitchEventBased
```

Comment:
This test didn't have the transition attribute? 😨

Reply:
I probably missed that =/
would be nice to add the assertions as well.

Reply:
Actually IIUC this PR is validating it. So we are fine.